authors/apps/usernotifications/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from ..authentication.models import User
+
+
+class Article(models.Model):
+    """This class creates the Article model"""
+    title = models.CharField(max_length=50, null=False, unique=True)
+    author = models.ManyToManyField(User)
+    description = models.CharField(max_length=250, null=False, default="")
+    body = models.TextField(null=False, default="")
+    read_time = models.PositiveIntegerField(default=1)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        """Returns a human readable representation of the model instance"""
+        return "{}".format(self.title)
+
+
+class Notification(models.Model):
+    """This class creates the Notification model"""
+    article = models.ManyToManyField(Article)
+    title = models.CharField(max_length=150, blank=False)
+    body = models.TextField()
+    user = models.ManyToManyField(User)
+    read = models.BooleanField(default=False)
```

Comment:
Great work! What's the essence of the read? What's the thought behind this boolean?

Reply:
@codeplus254  it tells truthy or falsy of whether the notification has been read